cmd/dagger/shell_commands.go

```diff
@@ -695,20 +695,21 @@ Without arguments, the current working directory is replaced by the initial cont
 		// TODO: Don't hardcode this list.
 		promoted := []string{
 			"address",
-			"llm",
 			"cache-volume",
-			"container",
 			"checks",
+			"container",
 			"directory",
 			"engine",
+			"env",
+			"env-file",
```

Comment:
turns out `env-file` was never added here. The rest of the changes are simply sorting the list alphabetically.